app/core/db.py

```diff
@@ -35,7 +40,10 @@ async def __call__(self) -> AsyncEngine:
         """Return an instance of AsyncEngine class."""
 
         if not self.instance:
-            self.instance = create_async_engine(ConfigClass.OPS_DB_URI)
+            try:
+                self.instance = create_async_engine(ConfigClass.OPS_DB_URI)
```

Comment:
You can also include this setting for future debugging purpose.
```suggestion
                self.instance = create_async_engine(ConfigClass.OPS_DB_URI, echo=ConfigClass.RDS_ECHO_SQL_QUERIES)
```